packages/slate/src/interfaces/editor.ts

```diff
@@ -1639,8 +1639,8 @@ export const Editor: EditorInterface = {
         continue
       }
 
-      if (node.text !== '' || Path.isBefore(path, blockPath)) {
-        end = { path, offset: node.text.length }
+      if (!Text.isText(node) || Path.isBefore(path, blockPath)) {
```

Comment:
Question related to the comment on line 1633 - what are we targeting here? An inclusive list might be better. Alternatively, we should extract a variable that describes the conditional.

Reply:
The conditions in the match function correspond to the following judgments. You're right, maybe we should revisit what does this function really want. Please do not deal with this PR for the time being. I will deal with the logic of this func again later if I have time

Reply:
Meaning you would like us to land the PR and then improve this, or meaning just wait until you have more time?

I think it's a pretty important fix given some of the recent regressions, so I'll take a look soon if you don't have time.

Reply:
I'm really sorry that I didn't have time to deal with these due to some unexpected things. I will close this PR first, and you can handle it first